src/utility/Parameters.cpp

```diff
@@ -21,28 +21,28 @@ Parameters *Parameters::getInstance() {
     return single;
 }
 
-// ========================================= Parse =======================================
-// ***************************************************************************************
+// ==================================== Parse ==================================
+// *****************************************************************************
 /// \brief  parses xml file
-/// \param	filename		string (name of xml-file)
-// ***************************************************************************************
+/// \param  filename        string (name of xml-file)
+// *****************************************************************************
 void Parameters::parse(const std::string& filename) {
     if (filename.empty()) {
-        std::cout << "no XML file specified, skip reading parameter" << std::endl;
+        // spdlog::error("no XML file specified, skip reading parameter");
```

Comment:
here should be some output

Reply:
> ah and I forgot: a few lines of output are still needed even in the benchmarking version, for example, the output of the global time. and maybe a line to inform the user that the simulation has started would be nice. Just to confirm that the software is calculating something. But I don't have a clean solution for that yet.

okay in that case I add some std::cout lines.

Reply:
fixed